EncoreTickets.SDK.Tests/UnitTests/Pricing/PricingServiceApiTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using EncoreTickets.SDK.Api.Models;
+using EncoreTickets.SDK.Api.Utilities.RequestExecutor;
+using EncoreTickets.SDK.Authentication;
+using EncoreTickets.SDK.Pricing;
+using EncoreTickets.SDK.Tests.Helpers.ApiServiceMockers;
+using NUnit.Framework;
+
+namespace EncoreTickets.SDK.Tests.UnitTests.Pricing
+{
+    internal class PricingServiceApiTests : PricingServiceApi
+    {
+        private MockersForApiServiceWithAuthentication mockers;
+
+        public override IAuthenticationService AuthenticationService => mockers.AuthenticationServiceMock.Object;
+
+        protected override ApiRequestExecutor Executor =>
+            new ApiRequestExecutor(Context, BaseUrl, mockers.RestClientBuilderMock.Object);
+
+        public PricingServiceApiTests() : base(new ApiContext(Environments.Sandbox))
+        {
+        }
+
+        [SetUp]
+        public void CreateMockers()
+        {
+            mockers = new MockersForApiServiceWithAuthentication();
+        }
+
+        [Test]
+        public void GetPriceBands_NullProductId_ArgumentException()
+        {
+            Assert.Catch<ArgumentException>(() => { GetPriceBands(null, 2, DateTime.Now); });
+        }
+    }
```

Comment:
Why are you ignoring unit tests for implemented methods except for one test above? Performing all unit tests is the first thing to check if something is not working in the project, and not integration tests